### PR TITLE
[infra/gbs] Build once on tizen test build

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates nnfw.spec rpm build script to remove duplicated build on tizen test build.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
cc/ @chunseoklee

**Review by chunseoklee — APPROVED** [review]

**Comment by hseok-oh** [comment]
@chunseoklee PTAL

I've build with no-test option, test option, and coverage option. But I didn't test package.

File: packaging/nnfw.spec [hunk 1]
```diff
@@ -125,22 +131,31 @@ NPU daemon for optimal management of NPU hardware
...
+
+# Set option for test build (and coverage test build)
+%define option_test -DENABLE_TEST=OFF
+%define option_coverage %{nil}
 %define test_suite_list infra/scripts tests/scripts
-%define test_build_type %{build_type}
+
+%if %{test_build} == 1
+# ENVVAR_ONERT_CONFIG: Use environment variable for runtime core configuration and debug
+%define option_test -DEANBLE_TEST=ON -DENVVAR_ONERT_CONFIG=ON
```
Comment (chunseoklee) [inline packaging/nnfw.spec:150]:
```suggestion
%define option_test -DENABLE_TEST=ON -DENVVAR_ONERT_CONFIG=ON
```

Comment (chunseoklee) [reply packaging/nnfw.spec:150]: @hseok-oh I found that nnfw-test build is done w/o this line patch. Any guess ?

Comment (hseok-oh) [reply packaging/nnfw.spec:150]:
```cmake
option(ENABLE_TEST "Build Tests using Google Test" ${GTest_FOUND})
```

If `test_build == 1`, build environment will install gtest, then `GTest_FOUND` will set. So `ENABLE_TEST` become `ON`.

**Comment by chunseoklee** [comment]
I have confirmed that test with cpu backend(since I have rpi3) works fine.

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**